src/documentSource.ts

```diff
@@ -1,8 +1,11 @@
-export enum DocumentSource {
-  DOCUMENT_SOURCE_TREEVIEW = 'treeview',
-  DOCUMENT_SOURCE_PLAYGROUND = 'playground',
-  DOCUMENT_SOURCE_COLLECTIONVIEW = 'collectionview',
-  DOCUMENT_SOURCE_CODELENS = 'codelens',
-}
+export const DocumentSource = {
+  DocumentSource_TREEVIEW: 'treeview',
+  DocumentSource_PLAYGROUND: 'playground',
+  DocumentSource_COLLECTIONVIEW: 'collectionview',
+  DocumentSource_CODELENS: 'codelens',
```

Comment:
```suggestion
  codelens: 'codelens',
```

If you are workshopping patterns it is also kind nice when the key and the value are identical. of course not always possible and I lean more towards less code changes than more when we just want to migrate off the code gen stuff. 

your upper snake case pattern is just as good, only sharing opinions here.